packages/SwingSet/README.md

```diff
@@ -53,7 +49,7 @@ In addition, a file named `bootstrap.js` must be present. This will contain
 the source for the "bootstrap Vat", which behaves like a regular Vat except:
 
 * At startup, its `bootstrap` method will be invoked, as `bootstrap(argv, vats)`
-* The `argv` value will be an array of strings, from the command line. So
+* The `argv` value will be an array of strings from the command line. So
```

Comment:
I think this is the only remaining case of `argv` actually being an array, along with some tests. Someday we'll clean this up I guess.